### PR TITLE
fix(implicit packages):Adding MAUI and C# markup packages when enabled

### LINKED ISSUE #18700: Adding Uno.Extensions.Maui.WinUI.Markup automatically if they use MAUI and C# markup
## What would you like to be added:
Adding Uno.Extensions.Maui.WinUI.Markup automatically automatically when you use MAUI Embedding and C# Markup

## Why is this needed:
Not to have to install it manually and deal with versions that can cause problems with Uno.Extensions.Maui.WinUI, and to be there since the beginning when creating a C# Markup project like other extensions

## For which Platform:

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renders for Xamarin.Forms
- [ ] Windows
- [x] Build tasks

## Anything else we need to know?
I can add it myself, I'll do it

### PR DESCRIPTION
**Pull request by LuisAlfredo92**
Also found a probable typo

GitHub Issue (If applicable): closes #18700

## PR Type

What kind of change does this PR introduce?
- Project automation

## What is the current behavior?
You have to add the NuGet package manually

## What is the new behavior?
It adds it automatically to the main project

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [x [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information
Number of tests passed and failed didn't change
imagen

Internal Issue (If applicable): #18700

### CODE REVIEW

**Review by MartinZikmund — APPROVED** [review]

**Comment by MartinZikmund** [comment]
@LuisAlfredo92 please make sure to reword the commits to adhere to Conventional Commits spec https://www.conventionalcommits.org/en/v1.0.0/

**Comment by LuisAlfredo92** [comment]
I did the change, is it correct now? Or do I have to create the fork again?

File: src/Uno.Sdk/targets/Uno.Implicit.Packages.ProjectSystem.targets [hunk 1]
```diff
@@ -44,6 +44,15 @@
 		<_UnoProjectSystemPackageReference Include="Uno.Extensions.Markup.Generators" ProjectSystem="true" />
 	</ItemGroup>
 
+	<ItemGroup Condition="$(UnoFeatures.Contains(';mauiembedding;'))">
```
Comment (jeromelaban) [inline src/Uno.Sdk/targets/Uno.Implicit.Packages.ProjectSystem.targets:47]: It feels like this section is not required, it's a duplicate of the other section in uno.extensions. It's likely, though, that the bug really was the typo that you found!

Comment (LuisAlfredo92) [reply src/Uno.Sdk/targets/Uno.Implicit.Packages.ProjectSystem.targets]: Done, it should work now

**Pull request merged**